code/modules/projectiles/guns/energy/energy.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/gun
-	name = "energy gun"
-	desc = "An E-07 energy gun manufactured by Shellguard Munitions. The fire selector has 'kill' and 'disable' settings."
+	name = "EG-7 energy gun"
```

Comment:
If you capitalize the starts of these names, you need to add `\improper` to ensure they're formatted right.

Without improper, this will be "You point at EG-7 energy gun.", vs "You point at the EG-7 energy gun."

Please apply this to all the gun-names that you've modified in this PR.
```suggestion
	name = "\improper EG-7 energy gun"
```